src/ResourceManager/Websites/Commands.Websites.Test/ScenarioTests/WebAppSlotTests.ps1

```diff
@@ -749,3 +749,81 @@ function Test-ManageSlotSlotConfigName
 	Assert-AreEqual 0 $slotConfigNames.AppSettingNames.Count
 	Assert-AreEqual 0 $slotConfigNames.ConnectionStringNames.Count
 }
+
+
+<#
+.SYNOPSIS
+Tests regular web app slot swap
+#>
+function Test-WebAppRegularSlotSwap
+{
+	$rgname = "Default-Web-EastAsia"
+	$appname = "webappslottest"
+	$sourceSlotName = "staging"
+	$destinationSlotName = "production"
+
+	# Swap Web App slots
+	$webApp = Swap-AzureRmWebAppSlot -ResourceGroupName $rgname -Name  $appname -SourceSlotName $sourceSlotName -DestinationSlotName $destinationSlotName
+}
+
+<#
+.SYNOPSIS
+Tests web app slot swap with preview: apply slot config followed by reset slot swap
+#>
+function Test-WebAppSwapWithPreviewResetSlotSwap
+{
+	Test-SlotSwapWithPreview 'ResetSlotSwap'
+}
+
+<#
+.SYNOPSIS
+Tests web app slot swap with preview: apply slot config followed by complete slot swap
+#>
+function Test-WebAppSwapWithPreviewCompleteSlotSwap
+{
+	Test-SlotSwapWithPreview 'CompleteSlotSwap'
+}
+
+<#
+.SYNOPSIS
+Test slot swap with preview feature
+#>
+function Test-SlotSwapWithPreview($swapWithPreviewAction)
+{
+    $rgname = "Default-Web-EastAsia"
```

Comment:
As the file is already using tabs instead of spaces, it's better to continue that coding style.